x-pack/plugins/security_solution/public/management/pages/trusted_apps/state/type_guards.ts

```diff
@@ -8,18 +8,18 @@
 import { ConditionEntryField } from '@kbn/securitysolution-utils';
 import type {
   TrustedAppConditionEntry,
-  MacosLinuxConditionEntry,
-  WindowsConditionEntry,
+  LinuxConditionEntry,
+  WindowsMacosConditionEntry,
 } from '../../../../../common/endpoint/types';
 
-export const isWindowsTrustedAppCondition = (
+export const includeSignerFieldCondition = (
```

Comment:
nit: these should be named like `isSignerFieldIncluded` and `isSignerFieldExcluded` to improve readability and showing that this is a query function, as the current name `includeSignerFieldCondition` sounds like a command

Reply:
Changed - bb014c4433bf1daec76cc5d61c00b26b75ed37c2